v22.2/backup-validation.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Backup Validation
+summary: Ensure that your backups are restorable with backup validation.
+toc: true
+docs_area: manage
+---
+
+{% include_cached new-in.html version="v22.2" %} CockroachDB provides backup validation tools to ensure that backups you have in storage are restorable. You can use the following option patterns with [`SHOW BACKUP`](show-backup.html) and [`RESTORE`](restore.html) to validate a [cluster](backup.html#backup-a-cluster), [database](backup.html#backup-a-database), or [table](backup.html#backup-a-table-or-view) backup. The three options result in increasing levels of backup validation:
+
+1. `SHOW BACKUP ... WITH check_files`: Check that all files belonging to a backup are in the expected location in storage. See [Validate backup files](#validate-backup-files) for an example.
```

Comment:
nit: looks like the numbering of this list isn't incrementing

Reply:
So, this looks good in the rendered preview. In markdown we add all numbered lists in this way. It kind of secures against if someone comes in and adds another item without updating the numbers as they will automatically render as incremented numbers. Thanks for checking though, because sometimes indents and stuff can mess that up.

Reply:
TIL